doc/content/integrations/mqtt/_index.md

```diff
@@ -32,7 +32,11 @@ You can now click on the **Generate new API key** button in order to generate an
 
 {{< figure src="mqtt-key-created.png" alt="MQTT API key created" >}}
 
-{{< note >}} Make sure to copy your API key now, since it will no longer be visible after leaving the page for security reasons. {{</ note >}}
+{{< note >}} 
+- Make sure to copy your API key now, since it will no longer be visible after leaving the page for security reasons.
+
+- If API key does not exist or has been removed, you will probably see the `as.mqtt.connect.fail` event with the `api key with id {api_key_id} not found` error in application Live events.
```

Comment:
```suggestion
If API key does not exist or has been removed, you will probably see the `as.mqtt.connect.fail` event with the `api key with id <api-key-id> not found` error in application Live events.
```

Reply:
> Make sure to copy your API key now, since it will no longer be visible after leaving the page for security reasons.

This is the current note point in the documentation. We added a new line specifically for the error. We believe that the point should be there. Otherwise, please let us know.

Reply:
Sorry, you're right, that part needs to stay. What I wanted to suggest was only to remove bullet points